### PR TITLE
add idle_timeout support

### PR DESCRIPTION
**Pull request by abridgett**
This adds idle_timeout support to ec2_elb_lb so that it can be changed from AWS's default 60seconds.

### CODE REVIEW

**Comment by robynbergeron** [comment]
Hi @abridgett -- thanks for submitting this PR.

@jsdalton & @rickmendes -- Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

Also pinging the other folks in the amazon module maintainer crew for good measure: @jsmartin @scicoin-project @tombamford @garethr @lorin @jarv @silviud @adq @zbal @zeekin @willthames @lwade @carsongee @defionscode @tastychutney @bpennypacker @loia

File: cloud/amazon/ec2_elb_lb.py [hunk 1]
```diff
@@ -102,6 +102,12 @@
     required: false
     aliases: []
     version_added: "1.8"
+  idle_timeout:
+    description:
+      - ELB connections from clients and to servers are timed out after this amount of time
+    required: false
+    aliases: []
```
Comment (ghost) [inline cloud/amazon/ec2_elb_lb.py]: not needed when there are no aliases - should be removed

**Comment by ghost** [comment]
Other than the documentation comment, this looks good to me. I have not had a chance to test it out yet.

**Comment by abridgett** [comment]
sorry, accidental closure.
Removed aliases as pointed out by @rickmendes

**Comment by abridgett** [comment]
Fixed merge conflict as devel has moved on since original submission.

**Comment by abridgett** [comment]
Just wondering if there was anything else I could do to assist as I realise this is going to keep drifting away from devel over time.

**Comment by ghost** [comment]
As no one else has commented on it in all of this time, I say shipit.

**Comment by gregdek** [comment]
Thanks @rickmendes for the review, and thanks @abridgett for the PR. Marking for potential inclusion.

**Comment by abridgett** [comment]
fantastic - many thanks!

**Comment by abadger** [comment]
# Merged

Hi!

This has been merged in, and will also be included in the next major release.

If you or anyone else has any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!

**Pull request merged**